apps/meteor/server/methods/registerUser.ts

```diff
@@ -129,6 +129,12 @@ Meteor.methods<ServerMethods>({
 			// throw new Meteor.Error 'error-email-send-failed', 'Error trying to send email: ' + error.message, { method: 'registerUser', message: error.message }
 		}
 
+		const defaultRoles = settings.get('Accounts_Registration_Users_Default_Roles');
+
+		if (!defaultRoles) {
+			await Users.updateOne({ _id: userId }, { $set: { roles: [] } });
```

Comment:
I'd suggest adding a new method to the `Users` model instead of calling this directly :thinking:

Reply:
We already have a method to insert a role to a user. We can use `addUserRoleAsync` or even `Roles.addUserRoles`; however, none of them allow us to insert an empty array of roles or a role with an invalid id. Since this is an isolated scenario and we, in fact, don't want to be this to become a common thing, I think that it's ok for us to update directly.
But please, share your thoughts if you disagree.